### ISSUE TITLE
No ./scripts or ./styles directories in build

### ISSUE THREAD

**Issue by shotlom** [choojs/bankai#242]
There is a mismatch between the references in build folder.  For example, index.html refers to

```html
    <script src="/scripts/69b4f9603b4b7c93/bundle.js" defer></script>
    <link rel="preload" as="style" href="/styles/f45e2938cc8b9aad/bundle.css" onload="this.rel='stylesheet'">
```
 However the build folder has only 

```
   69b4f9603b4b7c93/bundle.js
   f45e2938cc8b9aad/bundle.css
```

![image](https://user-images.githubusercontent.com/398402/30672024-472acb2e-9ead-11e7-8f07-9a5e5d8e82dc.png)

**Comment by yoshuawuyts** [comment]
Uh oh, good catch! Fixing it right away

**Comment by yoshuawuyts** [comment]
Fixed in 764c7e2 :tada:
